lib/sheltertech/db.rb

```diff
@@ -379,7 +407,12 @@ module Constants # rubocop:disable Metrics/ModuleLength
         'Re-entry Services',
         'Clean Slate',
         'Probation and Parole',
-        'MOHCD Funded Services'
+        'MOHCD Funded Services',
+        'Basic Needs & Shelter',
+        'Eviction Prevention',
```

Comment:
You can probably remove Eviction Defense, the category Eviction Prevention replaces. Also according to the requirements doc we are going to go with Housing instead of Affordable Housing:

https://sheltertech.quip.com/K0ZMA1vgbxjd/SFServiceGuideorg-Landing-Page-Requirements